src/wp-includes/media.php

```diff
@@ -1077,6 +1077,9 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 			}
 		}
 
+		// Add a `decoding` attribute.
+		$attr['decoding'] = 'async';
+
```

Comment:
This should be placed and similar to the setting of the default loading attribute on line 1048, ie use `$default_attr[decoding]` instead.

`decoding` should be added to the docblock too. 

Both of these are outside the diff, so I can't add line comments directly.

Reply:
👍🏼 will do, noticed that as well.

Reply:
Addressed in [`e3e7278` (#2409)](https://github.com/WordPress/wordpress-develop/pull/2409/commits/e3e72786ffd2c5e2e0968031989080a71e0260cb)